octopoes/octopoes/models/ooi/monitoring.py

```diff
@@ -19,6 +21,6 @@ class Incident(OOI):
     event_type: str
     event_title: str
     severity: str
-    meta_data: dict[str, Any]
+    meta_data: JsonValue
```

Comment:
Shouldn't this be the following like the change in config.py?

```suggestion
    meta_data: dict[str, JsonValue]
```

Reply:
I wasn't sure whether this field allows full JSON values or just a key-value/ object. I can change it to `dict[str, JsonValue]`?

Reply:
Implemented the suggestion, it seems indeed as if `Incident.meta_data` is filled with key-value pairs.